README.md

```diff
@@ -127,23 +134,30 @@ Following the example of a simple USER and ADMIN setup above, consider the follo
 					'*' => [
 						User::ROLE_ADMIN
 					],
-					
+
 					// here we explicitly list actions that
-					// USERS shall be able to access 
+					// USERS shall be able to access
 					'index' => [
 						User::ROLE_USER
 					],
 					'view' => [
 						User::ROLE_USER
-					]
+					],
+					// `AND` and `OR`
+					'delete' => [
+					    'AND' => [
```

Comment:
Why not doing it the same way as cake in the query conditions:
Elements in an array are linked by AND unless stated otherwise.